src/App.tsx

```diff
@@ -369,20 +367,17 @@ const App = (): JSX.Element => {
       }
 
       const newPrices = await response.json();
-      // Update the store only if the price is actually changed
-      if (
```

Comment:
note: I've removed this check, as we need to have tokens available by the CoinGecko name (i.e. polkadot, kintsugi) and these values will be different for interlay/kintsugi. Price data is going to be moved out of general state to a hook very soon